v3/src/data-interactive/handlers/attribute-handler.ts

```diff
@@ -1,42 +1,79 @@
+import { isAttributeType } from "../../models/data/attribute"
+import { withoutUndo } from "../../models/history/without-undo"
 import { getSharedCaseMetadataFromDataset } from "../../models/shared/shared-data-utils"
-import { DIHandler, DIResources, diNotImplementedYet } from "../data-interactive-types"
+import { t } from "../../utilities/translation/translate"
 import { registerDIHandler } from "../data-interactive-handler"
+import { DIHandler, DIResources, DIValues, diNotImplementedYet } from "../data-interactive-types"
 
+function convertAttributeToV2(resources: DIResources) {
+  const { attribute, dataContext } = resources
+  const metadata = (dataContext && getSharedCaseMetadataFromDataset(dataContext))
+  if (attribute) {
+    const { name, type, title, description, editable, id, precision } = attribute
+    return {
+      name,
+      type, // TODO This won't return "none", which v2 sometimes does
```

Comment:
I think this is fine. I looked into this recently and it seems like v2 used `none`/`null` interchangeably, so I think it's safe to eliminate `none` from consideration.